content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,120 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a
+[valid URL format](#url-format-to-specify-folder-location). Then use any DVC
+command that needs it (e.g. `dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d gdremote gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc push
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See the [Authorization](#authorization) section for more details on connecting
```

Comment:
this separate authorization still feels strange and not needed

Reply:
Well, the Authorization section is pretty long for the "Quick" start, and applies to any URL format and even with a custom GC project. I think that's why it's separate.

Reply:
In fact there's a link to it from the Custom GC project section.

Reply:
p.s. Also moved the Auth section back to the bottom since custom gc project and service acct sections are mentioned before quick start now, per https://github.com/iterative/dvc.org/pull/1096#pullrequestreview-387659149.